epochtalk/epochtalk
# Bans consolidation

Consolidates all banning methods/permissions into the bans model.

Test Procedure:
1) Attempt to ban a user via the Management->Users, Moderation->Users, Moderation->Posts and  Moderation->Messages Views. 

2) Test all ban types (Board Bans, Temporary Account Ban, Permanent Account Ban, and IP Account Ban)

3) Visit the Management->Roles view and attempt to change all ban related permissions in the "Views" and "Bans" tab. Ensure that the permissions stick and take appropriate affect when logging in as a user with the role with the modified ban permissions.

4) Ensure that all banning actions are showing up appropriately in the Moderation->Logs View. Also ensure filters for banning addresses show up in the drop down.
